jdbcSource/build.gradle

```diff
@@ -84,4 +84,18 @@ dependencies {
     // Used to create a connection pool if asynchronous processing has been specified for publish/query handlers
     compile group: 'com.zaxxer', name: 'HikariCP', version: '3.3.1'
 
+}
+
+// Using gradle-dcompose-plugin to build jdbcSource docker image
+dcompose {
+    jdbcSource {
+        buildFiles = project.copySpec {
+            from "$projectDir/src/main/docker/"
+            from "$projectDir/build/distributions/jdbcSource.tar"
+        }
+        repository = 'namirfa/namir-testing:jdbc'
```

Comment:
I think I'd prefer not to have this default to pushing to your repo.  If you merge, it goes into master so anyone playing with it...